aiosmtpd/controller.py

```diff
@@ -56,12 +64,17 @@ def start(self):
         self._thread.start()
         # Wait a while until the server is responding.
         ready_event.wait(self.ready_timeout)
-        if self._thread_exception is not None:
+        if self._thread_exception is not None:  # pragma: nowsl
+            # See comment about WSL1.0 in the _run() method
             raise self._thread_exception
 
     def _stop(self):
         self.loop.stop()
-        for task in asyncio.Task.all_tasks(self.loop):
+        try:
+            _all_tasks = asyncio.Task.all_tasks
+        except AttributeError:   # pragma: nocover
+            _all_tasks = asyncio.all_tasks
+        for task in _all_tasks(self.loop):
```

Comment:
Similar to #189 

The `pragma: nocover` is _at the moment_ needed to get 100% coverage. I'm trying to implement [`coverage-conditional-plugin`](https://github.com/wemake-services/coverage-conditional-plugin/) but had a glitch with that plugin earlier today. When I fixed the issue with that plugin, I'll replace `nocover` with something better.

Reply:
Good idea, I'm putting in the comment now, to go into the next commit round.